### PR TITLE
Rename azure-iot-manager to iot-manager

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

File: worker/update_device_status.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
                 "${workflow.input.azure_status}"
             ],
             "http": {
-                "uri": "http://${env.AZURE_IOT_MANAGER_ADDR|mender-azure-iot-manager:8080}/api/internal/v1/azure-iot-manager/tenants/${workflow.input.tenant_id}/bulk/devices/status",
+                "uri": "http://${env.IOT_MANAGER_ADDR|mender-iot-manager:8080}/api/internal/v1/iot-manager/tenants/${workflow.input.tenant_id}/bulk/devices/status",
```
Comment (merlin-northern) [inline worker/update_device_status.json:31]: the name of the workflow is still "update azure status" -- if it is on purpose, then ignore that comment.

**Review by merlin-northern — APPROVED** [review]
lgtm, one comment pre-approving.

File: worker/update_device_status.json (hunk 1 above)
Comment (tranchitella) [reply worker/update_device_status.json:31]: Yes, I think the name should stay "update azure status" as also the parameters are `azure_*`.
@alfrunes will change it once we start implementing additional cloud providers, I suppose.

**Pull request merged**